playground/signalr/SignalRServerlessWeb/Program.cs

```diff
@@ -1,9 +1,14 @@
+using System.Text.Json.Serialization;
 using Microsoft.AspNetCore.SignalR;
 using Microsoft.Azure.SignalR.Management;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Services.AddRazorPages();
+builder.Services.ConfigureHttpJsonOptions(options =>
+{
+    options.SerializerOptions.DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull;
```

Comment:
this changes the default behavior for all the apis. `var ingoreNullOption = new JsonSerializerOptions...` for negotiate is enough?